packaging/arch/PKGBUILD

```diff
@@ -3,28 +3,34 @@
 # Contributor: Zygmunt Krynicki <me at zygoon dot pl>
 
 pkgbase=snapd
-pkgname=(snapd snap-confine)
-pkgver=2.26.1
+pkgname=snapd
+pkgver=2.26.14.r1445.g3df03f98b
```

Comment:
Because this is a pre-release snapshot. Note that this version is automatically updated by `makepkg` due to `pkgver` function below.